src/contractWrappers/KlerosWrapper.js

```diff
@@ -382,16 +377,18 @@ class KlerosWrapper extends ContractWrapper {
       for (let appeal = 0; appeal <= numberOfAppeals; appeal++) {
         const voteCounts = []
         for (let choice = 0; choice <= rulingChoices; choice++)
-          voteCounts.push(contractInstance.getVoteCount(disputeId, appeal, choice).then(v => v.toNumber()))
+          voteCounts.push(
+            contractInstance
+              .getVoteCount(disputeId, appeal, choice)
+              .then(v => v.toNumber())
+          )
         voteCounters.push(voteCounts)
       }
-      [voteCounters, status] = await Promise.all(
-        [
-          Promise.all(voteCounters.map(voteCounts => Promise.all(voteCounts))),
-          contractInstance.disputeStatus(disputeId)
-        ]
-      )
-  
+      ;[voteCounters, status] = await Promise.all([
```

Comment:
extra semicolon or is this needed?

Reply:
Yes,

In general, `\n` ends a statement unless:

* The statement has an unclosed paren, array literal, or object literal or ends in some other way that is not a valid way to end a statement, (For instance, ending with . or ,.).
* The line is -- or ++, (in which case it will decrement/increment the next token.).
* It is a for(), while(), do, if(), or else, and there is no {.
* The next line starts with [, (, +, *, /, -, ,, ., or some other binary operator that can only be found between two tokens in a single expression.

The last one applies here so we need the explicit line break `;`.